src/styles/blocks/header.scss

```diff
@@ -0,0 +1,67 @@
+@import "../utils/variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  background: $white-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point